src/geo/ui/legends/custom-choropleth/legend-template.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="u-flex u-justifySpace u-bSpace--m">
+  <p class="CDB-Text CDB-Size-small"><%- prefix %> <%- suffix %></p>
```

Comment:
Aren't we missing something here between the prefix and the suffix? Take a look at [this](https://github.com/CartoDB/cartodb.js/blob/0d1d64e9666ce236dcab7b32e8e9c753090b6607/src/geo/ui/legends/choropleth/legend-template.tpl#L2)...